app/lib/dttp/access_token.rb

```diff
@@ -18,10 +15,8 @@ class Client
     }.freeze
 
     def self.fetch
-      Rails.cache.fetch("dttp-access-token", expires_in: CACHED_TOKEN_EXPIRY_TIME) do
-        response = Client.post("/#{Settings.dttp.tenant_id}/oauth2/v2.0/token", body: REQUEST_BODY)
-        JSON.parse(response.body)["access_token"]
-      end
+      response = Client.post("/#{Settings.dttp.tenant_id}/oauth2/v2.0/token", body: REQUEST_BODY)
```

Comment:
get a new token every time right?

Reply:
Yep. We decided that microsoftonline.com could probably handle the 10 tokens per week that we need at the moment and these are all called in Sidekiq jobs so we can handle failure and the extra request. We can revisit as our usage increases but the cacheing is causing us problems just now.